migrations/2021_03_16_171148_add_custom_command.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddCustomCommand extends Migration
+{
+	/**
+	 * Run the migrations.
+	 */
+	public function up()
+	{
+		Schema::table('schedules', function (Blueprint $table) {
```

Comment:
Here in the migration, the name of the table must use the field of the `config/database-schedule.php` configuration file